cmake/Dependencies.cmake

```
@@ -118,7 +118,7 @@ else()
   set(AT_MKLDNN_ENABLED 0)
   set(AT_MKL_ENABLED 0)
 endif()
-set_property(CACHE BLAS PROPERTY STRINGS "Eigen;ATLAS;OpenBLAS;MKL;vecLib;FLAME;Generic")
+set_property(CACHE BLAS PROPERTY STRINGS "Eigen;ATLAS;OpenBLAS;blis;MKL;vecLib;FLAME;Generic")
```

Is order important here? If not, can we just sort the alphabetically?
```suggestion
set_property(CACHE BLAS PROPERTY STRINGS "ATLAS;Eigen;FLAME;Generic;MKL;OpenBLAS;blis;vecLib")
```

---

Thanks for the suggestions, I've addressed these in the latest commit. Order is not important, sorted them alphabetically. Also changed blis to BLIS